tests/test_matrix_utils_auth_e2ee.py

```
@@ -20,6 +20,37 @@
 pytestmark = pytest.mark.asyncio
 
 
+def _matrix_capabilities(
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_matrix_capabilities`

(ANN202)